base/tools/src/main/java/com/netscape/cmstools/kra/KRAKeyFindCLI.java

```diff
@@ -95,6 +99,7 @@ public void execute(CommandLine cmd) throws Exception {
         String clientKeyID = cmd.getOptionValue("clientKeyID");
         String status = cmd.getOptionValue("status");
         String realm = cmd.getOptionValue("realm");
+        String ownerName = cmd.getOptionValue("ownerName");
```

Comment:
Here the value of the argument will be stored into the `ownerName` variable. So if you specify `--ownerName 10000000000000000002:foouser1` in the CLI, the `ownerName` variable will be `10000000000000000002:foouser1`.